http/unit-tests/CredentialsManagerTest.cc

```diff
@@ -125,17 +120,14 @@ class CredentialsManagerTest: public CppUnit::TestFixture {
             TheBESKeys::TheKeys()->set_key(CATALOG_MANAGER_CREDENTIALS, cm_config);
             CredentialsManager::theCM()->load_credentials();
         }
-        catch (BESError &e) {
+        catch (const BESError &e) {
```

Comment:
Probably there's a ticket here if `catch(const SomeError &e`) is the correct pattern. I doubt this happens in the BES's top level error handler or elsewhere in our code base.

Reply:
It is the correct pattern - or a 'better' pattern.